src/IssueViz.Security/HotspotsList/TableDataSource/NonNavigableFrameworkElementFactory.cs

```diff
@@ -0,0 +1,104 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Drawing;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Media;
+using Microsoft.VisualStudio.Shell.Interop;
+using Color = System.Windows.Media.Color;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource
+{
+    internal interface INonNavigableFrameworkElementFactory
+    {
+        FrameworkElement Create(string content);
+    }
+
+    internal class NonNavigableFrameworkElementFactory : INonNavigableFrameworkElementFactory
```

Comment:
I wanted to isolate the styling + vs theming from the standard HotspotTableEntry implementation that only handles texts.
I'm not sure though if it deserves a class of its own but it's got some meat to it, and I might need to tinker with it more in the future.
So right now I extracted it to another class but I don't know yet how it look like in the future. Let me know what you think.